src/Error/Error.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { screen, render } from '@testing-library/react';
+import Error from './Error';
+
+describe('Error', () => {
+  it('should render an error message', () => {
+    const mockError = 'Failed to Fetch';
+
+    render(
+      <Error
+        error={mockError}
+      />
+    )
+      const errorMessage = screen.getByText('Failed to Fetch');
+
+      expect(errorMessage).toBeInTheDocument();
+  })
```

Comment:
I am curious if we even need to test this. Or if there are better ways to handle testing for errors?